Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionWindow.mm

```diff
@@ -31,9 +31,11 @@
 #import "TestWebExtensionsDelegate.h"
 #import <WebKit/WKFoundation.h>
 #import <WebKit/_WKWebExtensionContextPrivate.h>
+#import <WebKit/_WKWebExtensionControllerPrivate.h>
```

Comment:
was getting build errors in TestWebKitAPI so I added these imports

Reply:
Yeah that happens when adding new files to the Sources.txt file. It means the file was missing the include but got it for free by being combined with another file before, and that file is no longer grouped with the same files due to the added file.